src/Microsoft.ML.Api/GeneratedCodeTemplate.csresource

```diff
@@ -45,7 +53,7 @@ PredictAsync(modelPath);
 /// This method demonstrates how to run prediction.
 ///
 ///</summary>
-public static void Predict(string modelPath)
+public static void PredictAsync(string modelPath)
```

Comment:
>PredictAsync [](start = 19, length = 12)

Did you try running this code now? I think the return type now should be `Task`?